src/code/z_play.c

```diff
@@ -1068,6 +1068,7 @@ void Play_Update(PlayState* this) {
 skip:
     PLAY_LOG(3801);
 
+    //! @bug If frame advancing or during tile transitions, isPaused will be used uninitialized.
```

Comment:
interesting, is this worth an `AVOID_UB`? This seems like something that could bite a modder when trying to use frame advance

Reply:
I'm not sure this behavior can reliably be made consistent since it uses an uninitialized value picked out of the stack (`sp + 0x70`)

Reply:
hmm, I was thinking of

```c
#ifdef AVOID_UB
// Uninitialized below blah blah blah
s32 isPaused = false;
#else
s32 isPaused;
#endif
```

but maybe that's not actually the intended behavior here?

Reply:
The intention of AVOID_UB is only to make behaviors consistent across compilers rather than to modify or fix behaviors. If we could find what value is on the stack at the time and that value is consistent and still accessible we could write an explicit assignment to it to make the behavior consistent

Reply:
hmm, it seems it's already used for bugfixes, for example here: https://github.com/zeldaret/oot/blob/cddca30442934fe48d31cadc2e626d7bd9253216/src/code/fault_gc.c#L374

this `isPaused` case is not important to me though so I'm fine doing nothing about it

Reply:
(never mind, I misread what was going on there)